### PR TITLE
more differentiation for Linux and Windows // -b parameter

### PR DESCRIPTION
**Pull request by mstenz**

### CODE REVIEW

**Comment by mstenz** [comment]
I have also removed the duplicate about "ascii character set"

File: end-user/en/source/pages/migration/scanning.rst [hunk 1]
```diff
@@ -7,17 +7,16 @@ Executing a Scan of the Source System
 
 The first step in migrating your system is running a scan of the target system. This identifies the meta-data of every file and package that makes up the running workload.
 
-You must have root access on the target system in order to complete the scan, as you will need to copy and run a binary file on the target system.
+You must have root (on Linux) or Administrator (on Windows) access on the target system in order to complete the scan, as you will need to copy and run a binary file on the target system.
 
-.. warning:: UForge AppCenter does not support multi-kernels. When scanning a machine with more than one kernel, only the kernel running will be scanned and imported.
+.. warning (Linux only):: UForge AppCenter does not support multi-kernels. When scanning a machine with more than one kernel, only the kernel running will be scanned and imported.
```
Comment (jgweir) [inline end-user/en/source/pages/migration/scanning.rst]: .. warning:: is actually a tag in rst.  You are breaking this tag by adding "(Linux only)" at this position.

I suggest the something like:

.. warning:: For Linux, UForge AppCenter does not support.....

File: end-user/en/source/pages/migration/scanning.rst [hunk 2]
```diff
@@ -42,6 +41,8 @@ To carry out a scan, go to the ``Migration`` tab:
 	9. Run the scan command on the running target environment to start a deep scan of the system you want to migrate. The binary identifies the packages, files and custom files on the system.
 
 		By default the scan data will be saved in ``/tmp``. You can modify the directory where the data will be saved using the ``-t`` option in order to ensure that there is enough space to save the scan data.
+		
+		By default all available bandwidth will be used. You can limit the bandwidth usage by using the ´´-b`` option following the reservation (for example ´´-b 50K`` to limit used bandwidth to 50 Kilobytes).
```
Comment (emuus) [inline end-user/en/source/pages/migration/scanning.rst]: Please make sure you are using the right quotes. Should be ``-b`` and ``-b 50K``.
Also I'm not sure what is meant by "following the reservation" ?

Comment (jgweir) [inline end-user/en/source/pages/migration/scanning.rst]: In .rst use the ``your text``, this effectively highlights it correctly.

**Pull request merged**